api/management/swagger-api_management.yaml

```diff
@@ -525,6 +525,49 @@ paths:
           description: "Invalid input"
         403:
           description: "No permission to call this operation"
+  /realms/{realm}/users/{userID}/send-reminder-email:
+    post:
+      tags:
+      - Users
+      summary: >
```

Comment:
You can remove ">" and merge lines 532 and 533

Reply:
Done